### ISSUE TITLE
Fern and alternate files <C-^>

### ISSUE THREAD

**Issue by oredaze** [lambdalisue/fern.vim#427]
I can't use <C-^> because it goes back to fern instead of the buffer that was active before fern was activated.
Is it possible to fix this?

**Comment by lambdalisue** [comment]
I'm sorry but I don't get your question. What is your expected behavior and how can I test or else...

**Comment by lambdalisue** [comment]
I cannot reproduce what you explained.

https://user-images.githubusercontent.com/546312/171682143-0a31538b-1656-46f1-a5c7-24c33c1e9b33.mp4

### Procedure

With Neovim 0.7.0, latest fern (master)

1. `:Fern .`
2. Select `LICENSE` and hit `<Return>`
3. `:Fern .`
4. Select `README` and hit `<Return>`
5. `<C-^>`
6. `<C-^>` ...

**Comment by oredaze** [comment]
Excellent. Works. I can't wait to play with fern.
Is there any reason why this is not the default?

**Comment by lambdalisue** [comment]
> Is there any reason why this is not the default?

Because it's ADDITIONAL behavior from the native Vim behavior.

**Comment by anordhoff** [comment]
Is it possible for the original edit buffer used to open Fern to remain as the alternative buffer, even if navigating around the project in Fern? I tried setting `let g:fern#keepalt_on_edit = 1` but it only seems to work if Fern is used to open a file in the initial directory, like the example you provided.

For example, given the following project structure
```
dir1
├── dir2
│   └── file2.txt
└── file1.txt
```

If I
1. nvim `file2.txt`
2. `:Fern %:h`
3. Navigate to the `dir1` directory using `<C-H>`
4: Select `file1.txt` and hit `<Return>`
5: Hit `<C-^>` to return to the alternative buffer

I expect the file2.txt buffer to be opened, but instead I'm returned to Fern with `dir1` opened, which was what I was given after step 2.

**Comment by lambdalisue** [comment]
I'm not sure but it sounds not supported because `<C-h>` invokes `leave` instead of `open:edit`.

Please create a new issue with more detail if you want that feature.
